pythonvectordbceph.py

```diff
@@ -144,6 +147,8 @@ def pythonvectordbappceph():
 
     app.logger.debug(vector)
     data = [ {"embedded_vector": vector, "url": object_url} ]
```

Comment:
this should be in the "else" part of the following "if"

Reply:
Sure will update